eclair-rpc/src/test/scala/org/bitcoins/eclair/rpc/EclairRpcTestUtilTest.scala

```diff
@@ -0,0 +1,91 @@
+package org.bitcoins.eclair.rpc
+import akka.actor.ActorSystem
+import akka.testkit.TestKit
+import org.bitcoins.eclair.rpc.client.EclairRpcClient
+import org.bitcoins.rpc.BitcoindRpcTestUtil
+import org.scalatest.{AsyncFlatSpec, BeforeAndAfterAll}
+
+import scala.collection.immutable.VectorBuilder
+
+class EclairRpcTestUtilTest extends AsyncFlatSpec with BeforeAndAfterAll {
+
+  private implicit val actorSystem: ActorSystem =
+    ActorSystem.create("EclairRpcTestUtilTest")
+
+  private val bitcoindRpc = BitcoindRpcTestUtil.startedBitcoindRpcClient()
+  private val clients: VectorBuilder[EclairRpcClient] =
+    new VectorBuilder[EclairRpcClient]
```

Comment:
`Vector.newBuilder[EclairRpcClient]` is the recommended way to create a new builder for any collection